digitalocean/provider.go

```diff
@@ -76,7 +76,7 @@ func Provider() *schema.Provider {
 			"http_retry_max": {
 				Type:        schema.TypeInt,
 				Optional:    true,
-				DefaultFunc: schema.EnvDefaultFunc("DIGITALOCEAN_HTTP_RETRY_MAX", 0),
+				DefaultFunc: schema.EnvDefaultFunc("DIGITALOCEAN_HTTP_RETRY_MAX", 4),
```

Comment:
We'll need to update the docs to reflect that as well: https://github.com/digitalocean/terraform-provider-digitalocean/blame/56ef0b601261e395a597ef7395c98cd44466a94c/docs/index.md#L69